packages/components/src/Components/ErrorState/ErrorState.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Stack, StackItem } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons/';
+import propTypes from 'prop-types';
+import './ErrorState.styles.scss';
+
+import {
+    Title,
+    EmptyState,
+    EmptyStateVariant,
+    EmptyStateIcon,
+    EmptyStateBody,
+    Button
+} from '@patternfly/react-core';
+
+const ErrorState = ({ errorTitle, errorDescription }) => {
+
+    const renderDescription = () => {
+        console.log('This is our error description: ', errorDescription);
+
+        if (errorDescription.length > 0) {
+            console.log('Rendering a description: ');
+
+            return (
+                <Stack>
+                    <StackItem>
+                        {errorDescription}
+                    </StackItem>
+                </Stack>
+            );
+        }
+
+        return (
+            <Stack>
+                <StackItem>
+                    There was a problem accessing the request. Please try again.
+                </StackItem>
+                <StackItem>
+                    If the problem persists, contact <a href="https://access.redhat.com/support?extIdCarryOver=true&sc_cid=701f2000001Css0AAC">
+                    Red Hat Supprt</a> or check out our <a href="status.redhat.com"> status page</a> for known outages.
+                </StackItem>
+            </Stack>
+        );
+    };
+
+    return (
+        <EmptyState variant={EmptyStateVariant.large}>
+            <EmptyStateIcon icon={ExclamationCircleIcon} />
+            <Title headingLevel='h4' size='lg'>
+                {errorTitle}
+            </Title>
+            <EmptyStateBody>
+                <>
+                    {renderDescription()}
```

Comment:
This function looks a bit complicated, how about this approach
```JSX
<EmptyStateBody>
<Stack>
{ !errorDescription && (
  <StackItem>
    There was a problem accessing the request. Please try again.
  </StackItem>
)}
<StackItem>
  { errorDescription || <DefaultMessage /> }
</StackItem>
</Stack>
</EmptyStateBody>
```

Where `DefaultMessage` would be const above this component
```JSX
const DefaultMessage = () => (
  <React.Fragment>
    If the problem persists, contact <a href="https://access.redhat.com/support?extIdCarryOver=true&sc_cid=701f2000001Css0AAC">
    Red Hat Supprt</a> or check out our <a href="status.redhat.com"> status page</a> for known outages.
  </React.Fragment>
);
```

Reply:
Changed to cleaner logic, thanks for the explanation!